middlewares/stripPrefix.go

```diff
@@ -18,6 +19,7 @@ type StripPrefix struct {
 func (s *StripPrefix) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	for _, prefix := range s.Prefixes {
 		if p := strings.TrimPrefix(r.URL.Path, strings.TrimSpace(prefix)); len(p) < len(r.URL.Path) {
+			log.Debugf("Striping Path Prefix %s from %s = %s", prefix, r.URL.Path, p)
```

Comment:
Could you remove this Debug log? This is way too verbose to keep it in production code :)